lib/instance-validator.js

```diff
@@ -108,7 +108,7 @@ InstanceValidator.prototype._builtinValidators = function() {
   var validators = [];
   Utils._.forIn(this.modelInstance.rawAttributes, function(rawAttribute, field) {
     if (self.options.skip.indexOf(field) >= 0) {
-      return;
+      return false;
```

Comment:
It was exiting early with just "return", but the lodash docs state that it should be [return false](https://lodash.com/docs#forIn). I changed this line just so that it was consistent with my main changes.